cluster/bin/snappy-job.sh

```diff
@@ -119,6 +120,15 @@ while (( "$#" )); do
       shift
       contextName="${1:-$TOK_EMPTY}"
     ;;
+    --user)
+      shift
+      username="${1:-$TOK_EMPTY}"
+      while [ -z ${password} ]; do
+        read -s -p "Enter password for ${username}: " password
+        echo ""
+      done
+      securePart="-u ${username}:${password}"
+    ;;
```

Comment:
@ashetkar Users can object to the fact that password will be sent as an argument to the command below and hence will be visible in the OS process list (even those belonging to other users). Need to discuss how this can be improved. I guess need to use a client like: https://github.com/bluebreezecf/SparkJobServerClient and allow it to read credentials from a properties file or something.

Please track this generic requirement as a task and list this under it. The task being "Secure password handling and shipping throughout the system": a) properties files for all components, b) key pair to encrypt/decrypt